website/pages/docs/commands/index.mdx

```diff
@@ -288,6 +288,12 @@ this environment variable is most useful when using the Go
 The namespace to use for the command. Setting this is not necessary
 but allows using relative paths.
 
+### `VAULT_SRV_LOOKUP`
+
+Enables the client to lookup the host through DNS SRV lookup as described in this
```

Comment:
I think you should also mention it is also necessary that they don't specify a port otherwise it will not work.